entry_types/scrolled/package/src/frontend/MediaPlayer/index.js

```diff
@@ -0,0 +1,42 @@
+import React, {useEffect, useRef} from 'react';
+import {media} from 'pageflow/frontend';
+
+import ScrollToSectionContext from "../ScrollToSectionContext";
+
+export function MediaPlayer(props){
+  const playerWrapperRef = useRef(null);
+  let scrollToSection = undefined;
+
+  useEffect( () => {
+    let playerWrapper = playerWrapperRef.current;
+    
+    if (props.sources) {
+      let player = media.getPlayer(props.sources, {
+        poster: props.poster,
+        playsInline: props.playsInline,
+        loop: props.loop,
+        controls: props.controls,
+        tagName: props.type,
+        ended: function () {
+          props.nextSectionOnEnd && scrollToSection('next');
+        }
```

Comment:
If you listen to this event via `player.on`, you can get around the special handling for the `ended` option in `createMediaPlayer`.

Reply:
I did try this initially but due to some reason event was not getting invoked at that time but now I have tried it again and it is working fine.